drone/exec.go

```diff
@@ -141,6 +149,7 @@ func execCmd(c *cli.Context) error {
 
 	var exits []int
 
+axes:
```

Comment:
Yes, I think I have seen this being done in the stdlib a number of times (I could be wrong). Anyway it's just the Go early return philosophy, I even thnk this is mentioned inneffective Go. The label is there for making the code easier to read.